### PR TITLE
fix(css): extract modal resets from elements reset

### PR DESCRIPTION
**Pull request by alexkrolick**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @alexkrolick has signed the CLA. Thanks for the pull request!

File: src/elements/common/base.scss [hunk 1]
```diff
@@ -70,6 +54,32 @@
     }
 }
 
+.be {
+    @include be-content;
```
Comment (jstoffan) [inline src/elements/common/base.scss]: It looks like most of the rules in `be-content` were declared after the `@import` statements below. I'd suggest splitting `@include reset` out:

```
.be {
    @include reset;
    
    // links have tag styles
    @import '../../styles/common/links';
    // Forms have tag styles
    @import '../../styles/common/forms';

    @include be-content;
    // etc.
}
```

**Pull request merged**